packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -947,24 +948,66 @@ abstract class ResidentRunner {
   void printHelp({ @required bool details });
 
   void printHelpDetails() {
+    if (flutterDevices.any((FlutterDevice d) => d.device.supportsScreenshot)) {
+      printCommandHelp(
+          's', 'Save a screenshot to flutter.png.');
```

Comment:
Please see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting for the overall rules for formatting in this repo. We try to avoid egregious issues that dartfmt introduces, such as this kind of wrapping.

You can just put this all on one line. The same applies to most of the cases below.